### PR TITLE
M3-4701: Move theme toggle to My Profile > Settings

### PR DESCRIPTION
**Pull request by DevDW**
## Description
This PR moves the theme toggle to the My Profile > Settings page.

- The now-unnecessary related code has been removed from `PrimaryNav_CMR.tsx`
- New `ThemeToggle_CMR.tsx` file created since the inline "Light" and "Dark" labels have been removed
- Theme toggle added to Settings page, based on `flags.cmr`

## Type of Change
- Non breaking change ('update', 'change')

## To-Do:
- [x] Theme toggle functionality, label logic

### CODE REVIEW

**Comment by DevDW** [comment]
There's a slight delay in the text next to the toggle button updating -- looking into that

File: packages/manager/src/features/Profile/Settings/Settings.tsx [hunk 1]
```diff
@@ -56,38 +72,61 @@ class ProfileSettings extends React.Component<CombinedProps, State> {
...
+            <PreferenceEditor
+              open={this.state.preferenceEditorOpen}
+              onClose={() => this.setState({ preferenceEditorOpen: false })}
+            />
+          )}
+        </Paper>
+        {flags.cmr ? (
+          <Paper className={classes.root}>
+            <Typography variant="h2" className={classes.title}>
+              Dark Mode
```
Comment (Jskobos) [inline packages/manager/src/features/Profile/Settings/Settings.tsx:111]: Is it officially called Dark Mode? I sort of like "Theme", with Light and Dark being the choices.

File: packages/manager/src/features/Profile/Settings/Settings.tsx [hunk 2]
```diff
@@ -56,38 +72,61 @@ class ProfileSettings extends React.Component<CombinedProps, State> {
...
+            </Typography>
+            <Grid container alignItems="center">
+              <Grid item xs={12}>
+                <FormControlLabel
+                  className="toggleLassie"
```
Comment (Jskobos) [inline packages/manager/src/features/Profile/Settings/Settings.tsx:116]: might want to rename this class

File: packages/manager/src/features/Profile/Settings/Settings.tsx [hunk 3]
```diff
@@ -56,38 +72,61 @@ class ProfileSettings extends React.Component<CombinedProps, State> {
...
+                  control={<ThemeToggle_CMR toggleTheme={toggleTheme} />}
+                  label={`
+                Dark mode is ${
+                  preferences.theme === 'dark' ? 'enabled' : 'disabled'
```
Comment (Jskobos) [inline packages/manager/src/features/Profile/Settings/Settings.tsx:120]: The way to remove the delay is to import `withTheme` in this component and make this conditional on `this.props.theme.name === 'darkTheme'`

**Review by Jskobos — COMMENTED** [review]
This looks good, see comments about resolving the delay and then should be good to merge.

**Review by tiffwong — APPROVED** [review]
Other than the delay, everything else looks good!

File: packages/manager/src/features/Profile/Settings/Settings.tsx (hunk 2 above)
Comment (Jskobos) [reply packages/manager/src/features/Profile/Settings/Settings.tsx:116]: turns out this is a non-MUI classname that's never actually defined anywhere. Probably been dead code for years.

**Review by Jskobos — APPROVED** [review]

**Pull request merged**